utils/ALA_dialogues.cfg

```diff
@@ -365,11 +365,11 @@
     {MOVE_UNIT (id=Divaid,Croquet) 8 13}
     [message]
         speaker=Glauco
-        message= _ "What's the situation, "+$trapper_name+"?"
+        message= _ "What's the situation?"
     [/message]
-    {SCOUT_MESSAGE ( _ "I would say it's bad, but I'm afraid it's awful; The old caravan station was partially swallowed up by the river, and it seems that a group of Nagas have nestled there.")}
+    {SCOUT_MESSAGE ( _ "I would say it's bad, but I'm afraid it's awful. The old caravan station was partially swallowed up by the river, and it seems that a group of Nagas have nestled there.")}
     {SCROLL_AND_DELAY 30 7 1500}
-    {SCOUT_MESSAGE ( _ "Also, on the hills just to the south there are small buildings, and from the stench that comes out I would say they are Ogre huts.")}
+    {SCOUT_MESSAGE ( _ "Also, on the hills just to the south of there are small buildings, and from the stench that comes out I would say they are Ogre huts.")}
```

Comment:
"of" is unnecessary here; it could also be replaced by putting a comma after "south"

Reply:
Yes. So use either coma or of.

Reply:
I went with a comma in [55af1f6](https://github.com/cooljeanius/A_Little_Adventure/pull/23/commits/55af1f6dfc81062b9643144602d6a4a28c4b3324)